pkg/iac/scanners/terraform/parser/load_blocks.go

```diff
@@ -1,131 +0,0 @@
-package parser
```

Comment:
where's this logic been moved to?

Reply:
I put the parsing of comments with ignore rules in a separate package https://github.com/aquasecurity/trivy/pull/6358/files#diff-c1fe13f24e97f852991501a2535948a2ff13ef4e36564313e85754d8f9c6f35eR20.